Predavanje trece zadatak

Zokky2e opened the pull request:
First version of configuration

P.S. nisam bio na predavanju

Zokky2e committed:
feature commit

Zokky2e committed:
commit 1

first version of the pizz-a-tron configurator page

Zokky2e committed:
added functions of changing price of pizza

depending of toppings and/or size of pizza change the price of the pizza that you are ordering

lbuljan reviewed (changes requested):
Looks good when run, but there are a couple of caveats I'd like you to change.

1. Folder naming is always lowercase with a dash if there are spaces. For instance `components/list-item`, not `Components/ListItem`
2. You did not create modules at all. The correct folder structure would be:
          modules/configurator/
          ->/components
          ->/context
          index.ts
       
 3. You did not separate components into their own folders.
 4. It's not as big of a deal, but I'd recommend using emotion css: https://emotion.sh/docs/introduction

Zokky2e committed:
reordered folder-file structure

as asked in last pull request I reordered the folder structure, but I didnt add emotion css

Zokky2e commented:
> Looks good when run, but there are a couple of caveats I'd like you to change.
> 
> 1. Folder naming is always lowercase with a dash if there are spaces. For instance `components/list-item`, not `Components/ListItem`
> 2. You did not create modules at all. The correct folder structure would be:
>    modules/configurator/
>    ->/components
>    ->/context
>    index.ts
> 3. You did not separate components into their own folders.
> 4. It's not as big of a deal, but I'd recommend using emotion css: https://emotion.sh/docs/introduction

I restructured the folder structure as you suggested, only thing I didnt redo was the emotion css adition, in bigger projects I tend to use sass and order it in smaller files without reusing too much css. So adding emotion would clash with my sass modules.
